modules/schain.js

```diff
@@ -197,11 +202,15 @@ init()
 
 export function setOrtbSourceExtSchain(ortbRequest, bidderRequest, context) {
   if (!deepAccess(ortbRequest, 'source.ext.schain')) {
-    const schain = deepAccess(context, 'bidRequests.0.schain');
+    // Look for schain in the new location: ortb2.source.schain
+    const schain = deepAccess(context, 'bidRequests.0.ortb2.source.schain');
```

Comment:
If you use `enrichFPD` as I suggest above, it would already be populated in `ortbRequest` here - but to keep the same behavior this would then need to move it from `source.schain` to `source.ext.schain`.